b2b-sample/apps/business-admin-app/pages/api/settings/user/viewUsers.ts

```diff
@@ -40,7 +40,7 @@ export default async function viewUsers(req: NextApiRequest, res: NextApiRespons
 
     try {
         const fetchData = await fetch(
-            `${getOrgUrl(orgId)}/scim2/Users`,
+            `${getOrgUrl(orgId)}/scim2/Users?count=100`,
```

Comment:
Fix for users not listing the sample app. Without a count, the SCIM user list API response will be in below format.
```
{
    "totalResults": 1,
    "startIndex": 1,
    "itemsPerPage": 0,
    "schemas": [
        "urn:ietf:params:scim:api:messages:2.0:ListResponse"
    ]
}```

Reply:
Shall we add `domain=<userstore>` attribute if there is a defined userstore here https://github.com/wso2/samples-is/pull/405/files#diff-34fe1b8cbee00497c7574755a2a49dc01937e2a04b08a16c2e7d3cfd24822197R130

Reply:
Adding count is not necessary right. IMO, looks like the SCIM api is broken

Reply:
Need to fix the SCIM API. Also need to add the https://github.com/wso2/samples-is/pull/405#discussion_r1146177770 improvement